package.json

```diff
@@ -16,7 +16,6 @@
   "dependencies": {
     "bcrypt-nodejs": "0.0.3",
     "body-parser": "^1.17.1",
-    "colored-console":"^0.10.0",
```

Comment:
Please rerun `yarn` so that `colored-console` will be removed from the `yarn.lock` file.